swagger/swagger-generator/generator-springmvc/src/main/java/org/apache/servicecomb/swagger/generator/springmvc/processor/annotation/RequestParamAnnotationProcessor.java

```diff
@@ -32,4 +33,13 @@ protected QueryParameter createParameter() {
   protected String getAnnotationParameterName(Object annotation) {
     return ((RequestParam) annotation).name();
   }
+
+  @Override
+  protected String getAnnotationParameterDefaultValue(Object annotation) {
+    String defaultValue = ((RequestParam) annotation).defaultValue();
+    if (defaultValue.equals(ValueConstants.DEFAULT_NONE)) {
+      return "";
```

Comment:
maybe null is better?

Reply:
might cause NullPointerException, also some message might get added with null which may not be correct , so returning empty

Reply:
Maybe it's better to return ValueConstants.DEFAULT_NONE. When no default value specified and users do not give the parameter value, we can throw an BadRequest exception.